Dockerfile.tracee-make

```diff
@@ -0,0 +1,61 @@
+# Instructions on how to use docker to build tracee:
+#
+# To create 'tracee-make' container:
+#
+# docker build -f Dockerfile.tracee-make \
+#       -t tracee-make:latest \
+#       --build-arg uid=$UID \
+#       --build-arg gid=$GID \
+#       .
+#
+# To exec the 'tracee-make' shell:
+#
+# docker run --rm --pid=host --privileged \
+#       -v $(pwd):/tracee \
+#       --entrypoint=/bin/bash \
+#       -it tracee-make
+#
+# tracee@xxxx[/tracee]$ make -f Makefile.one clean
+# tracee@xxxx[/tracee]$ make -f Makefile.one all
+# tracee@xxxx[/tracee]$ sudo ./dist/tracee-ebpf --debug --trace 'event!=sched*'
+#
+# To use 'tracee-make' as replacement for the make tool:
+#
+# docker run --rm --pid=host --privileged \
+#       -v $(pwd):/tracee \
+#       -it tracee-make \
+#       make -f Makefile.one [clean|core|all|test|...]
+#
+
+FROM alpine:3.15 as tracee-make
+
+ARG uid=1000
+ARG gid=1000
+
+RUN apk --no-cache update && \
+    apk --no-cache add sudo coreutils bash git curl && \
+    apk --no-cache add musl-dev libc6-compat && \
+    apk --no-cache add llvm clang go make gcc && \
+    apk --no-cache add linux-headers && \
+    apk --no-cache add elfutils-dev && \
+    apk --no-cache add libelf-static && \
+    apk --no-cache add zlib-static
+
+RUN curl -L -o /usr/bin/opa \
+    https://github.com/open-policy-agent/opa/releases/download/v0.33.1/opa_linux_amd64_static && \
+    chmod 755 /usr/bin/opa
+
+RUN export uid=$uid gid=$gid && \
+    mkdir -p /home/tracee && \
+    echo "tracee:x:${uid}:${gid}:Tracee,,,:/home/tracee:/bin/bash" >> /etc/passwd && \
+    echo "tracee:x:${gid}:" >> /etc/group && \
+    echo "tracee ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/tracee && \
+    chmod 0440 /etc/sudoers.d/tracee && \
+    chown ${uid}:${gid} -R /home/tracee && \
```

Comment:
Why would this brake the case you brought ? I'm just making sure that the tracee-make container uses the same UID as your host environment (because of generated files permissions). This way, all generated files will have same UID/GID as they would have in the host env (which is something we want).

Let me know if you find a use case we are breaking, I would like to consider it.

Reply:
maybe break was overreacting :) generating files owned by another user might create issues if the host user want's to manipulate the file (even just delete it). not a big deal, I'm resolving this